docs/database/sql-server-components.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Connect an ASP.NET Core app to SQL Server using .NET Aspire and Entity Framework Core
```

Comment:
This title will be truncated. You can use the `Learn: Search Results Preview` command in VSCode to see the results. Also, this wouldn't include your title postfix from the docfx file. Here's what I see:

![image](https://github.com/dotnet/docs-aspire/assets/67293991/3bc79078-f76b-4859-b4aa-b21520667d30)